### PR TITLE
Add switch_completed callback

### PR DESCRIPTION
**Pull request by Keshoid**
Add switch_completed callback.
- switch_completed is called after all actions are shown in current context. Useful for debot browser that accumulate actions before showing them to the user.
- Removed adapter.rs. It is not used.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ All notable changes to this project will be documented in this file.
 ### New
 - GraphQL optimization: use single web socket to serve all subscriptions.
 - `net.find_last_shard_block` function returning account shard last block ID.
+- **Debot Module:**
+  - Add new browser callback `switch_completed` to notify browser when all context actions are shown.
+  - `adapter.rs` was removed.
```
Comment (AlexeyVavilin) [inline CHANGELOG.md:11]: Not useful for user. Remove

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ All notable changes to this project will be documented in this file.
 ### New
 - GraphQL optimization: use single web socket to serve all subscriptions.
 - `net.find_last_shard_block` function returning account shard last block ID.
+- **Debot Module:**
+  - Add new browser callback `switch_completed` to notify browser when all context actions are shown.
```
Comment (AlexeyVavilin) [inline CHANGELOG.md:10]: Mention new enum variant for this callback

**Review by AlexeyVavilin — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Keshoid) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (Keshoid) [reply CHANGELOG.md]: Fixed

**Pull request merged**